boards.txt

```diff
@@ -25172,3 +25396,4 @@ namino_arancio.menu.EraseFlash.all=Enabled
 namino_arancio.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+>>>>>>> 057eac66472d8491fc2bcd1c49e8bcdbce59a75b
```

Comment:
Remove this. It's some leftover from the conflicts.
```suggestion
```

Reply:
resolved